bucket/schemacrawler.json

```
@@ -0,0 +1,17 @@
+{
+  "version": "16.16.5",
+  "description": "SchemaCrawler - Free database schema discovery and comprehension tool",
```

```suggestion
  "description": "Database schema discovery and comprehension tool",
```
No need to repeat name in description.

---

I have removed the product name from the description but left the tag line is intact.